docs-chef-io/content/server/reusable/md/server_services_redis.md

```diff
@@ -1,2 +1,6 @@
 Key-value store used in conjunction with Nginx to route requests and
 populate request data used by the Chef Infra Server.
+
+Note: From Chef Infra Server 15.10.x onwards, KeyDB service replaces Redis.
```

Comment:
**Replace it with:** {{< note >}} From Chef Infra Server 15.10. x onwards, the [KeyDB] (https://docs.keydb.dev/docs/about) service replaces Redis. The configuration name and service name will continue to be `redis_lb`. {{< note/ >}}